src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -247,6 +272,44 @@ public synchronized ProcessWrapper launchTranscode(
 		return pw;
 	}
 
+	/**
```

Comment:
Since these are already in `FFmpegVideo` which this extends, we shouldn't need to add them again unless they are different from those ones, in which case the `@override` flag should be present